exist-parent/pom.xml

```diff
@@ -560,19 +560,19 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-clean-plugin</artifactId>
-                    <version>3.3.2</version>
+                    <version>3.4.0</version>
                 </plugin>
                 <plugin>
                     <groupId>com.mycila</groupId>
                     <artifactId>license-maven-plugin</artifactId>
-                    <version>4.3</version>
+                    <version>4.5</version>
                 </plugin>
                 <plugin>
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
-                    <version>9.0.9</version>
+                    <version>10.0.3</version>
                     <configuration>
-                        <nvdApiKey>${env.NVD_API_KEY}</nvdApiKey>
+                        <nvdApiKey>cb452226-8de1-4d27-9032-7c9b8741c880</nvdApiKey>
```

Comment:
Should this be hardcoded?

Reply:
I tried and tried and tried.... it should be injected, which is working for develop, but I spent hours on the develop-6.x.x branche, nothing worked. I hardcoded it, because it was the only way to get it working.

The key is not really sensitive, so it should be ok-ish

Reply:
I see @dizzzz  could you open an issue and assign it to me. I'm afk for a while and would like to gave a go at it once I m back. Should be ok to leave this for a limited time.

Reply:
done. thnx (CC: @reinhapa )